### PR TITLE
172 skins

### LINKED ISSUE #172: Adding more skins
Adding more skins, based on the looks of the members of the software engineering group.

 - [x]  Bacchelli
 - [x]  Erik Meijer
 - [x]  Arie
 - [x]  Rini
 - [x] Guido

### PR DESCRIPTION
**Pull request by MKerkhof**
Will close #172

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/shop/ShopItemLoader.java [hunk 1]
```diff
@@ -2,13 +2,7 @@
...
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
-import nl.tudelft.ti2206.group9.shop.skin.AndySkin;
-import nl.tudelft.ti2206.group9.shop.skin.BoySkin;
-import nl.tudelft.ti2206.group9.shop.skin.CaptainSkin;
-import nl.tudelft.ti2206.group9.shop.skin.IronManSkin;
-import nl.tudelft.ti2206.group9.shop.skin.NoobSkin;
-import nl.tudelft.ti2206.group9.shop.skin.PlankSkin;
-import nl.tudelft.ti2206.group9.shop.skin.AbstractSkin;
+import nl.tudelft.ti2206.group9.shop.skin.*;
```
Comment (MathiasMeuleman) [inline src/main/java/nl/tudelft/ti2206/group9/shop/ShopItemLoader.java]: Please avoid star-imports
<review

**Comment by MathiasMeuleman** [comment]
I reviewed your changes. Had one comment, please change that, then this is ready to be merged. 

<review

**Comment by MKerkhof** [comment]
Fixed SAT warnings, forgot about them. Should be ready to merge now.

<review

**Comment by mdingjan** [comment]
I've fixed the last PMD warning and reviewed the changes.
Maybe it's useful to keep the prices of the skins in order; from low to high.
Could you fix this last thing?

After that, ready for merging!

<review,feature

**Comment by mpsijm** [comment]
Skins are looking very good, our teachers can be proud to be in our game :D
I fixed your final SAT warnings, will merge once Travis is happy :D

<review, layout, feature

**Pull request merged**